examples/authorization_details_mso_doc.json

```diff
@@ -2,15 +2,23 @@
   {
     "type": "openid_credential",
     "credential_configuration_id": "org.iso.18013.5.1.mDL",
-    "claims": {
-      "org.iso.18013.5.1": {
-        "given_name": {},
-        "family_name": {},
-        "birth_date": {}
+    "claims": [
+      {
+        "namespace": "org.iso.18013.5.1",
```

Comment:
this is missing path and does not match the claims description further down

Reply:
made a suggestion below